cli/Makefile

```diff
@@ -115,7 +117,7 @@ publish: clean prepublish build
 	cd $(CLI_DIR)/../packages/create-turbo && npm publish --tag $(TURBO_TAG)
 	cd $(CLI_DIR)/../packages/turbo-codemod && npm publish --tag $(TURBO_TAG)
 
-	git push origin main "v$(TURBO_VERSION)"
+	git push -f origin main "v$(TURBO_VERSION)"
```

Comment:
Is this necessary?

Reply:
Also do you need to push up the tag?

Reply:
1. `--force` is necessary as `main` is a protected branch—that's also why there is a `TURBOBOT` token involved here. I explicitly elected _against_ `--force-with-lease` because if we're running a release it's extremely important for the actual commit that the release is based on (hash) to be on `main`. In other words: if this clobbers something else then we fix the _something else_, not the release.
2. By specifying the "v$(TURBO_VERSION)" the tag comes along for the ride.